www/src/pages/components/table.js

```diff
@@ -16,22 +17,39 @@ export default function TableSection({ data }) {
       </h2>
 
       <p>
-        Use the <code>striped</code>, <code>bordered</code>,{' '}
-        <code>condensed</code> and <code>hover</code> props to customise the
-        table.
+        Use the <code>striped</code>, <code>bordered</code>, <code>sm</code>,{' '}
+        <code>hover</code> and <code>dark</code> props to customise the table.
       </p>
       <ReactPlayground codeText={TableBasic} />
 
       <h2>
         <Anchor id="table-responsive">Responsive</Anchor>
       </h2>
       <p>
-        Add <code>responsive</code> prop to make them scroll horizontally up to
-        small devices (under 768px). When viewing on anything larger than 768px
-        wide, you will not see any difference in these tables.
+        Responsive tables allow tables to be scrolled horizontally with ease.
+      </p>
+
+      <h3>
+        <Anchor id="table-responsive">Always Responsive</Anchor>
+      </h3>
+      <p>
+        Across every breakpoint, use <code>responsive</code> for horizontally
+        scrolling tables.
```

Comment:
lets maybe add a Callout as well here that responsive tables are wrapping in another div